authors/apps/notifications/utils.py

```diff
@@ -11,6 +13,24 @@
 
 domain = os.getenv('DOMAIN')
 
+opt_url = "{}/api/notifications/subscriptions".format(domain)
+
+
+class PageNumberPaginationNotifications(PageNumberPagination):
```

Comment:
Good work in implementing pagination for notifications data. LGTM

Reply:
Thank you